runtime/src/lib.rs

```diff
@@ -526,12 +528,22 @@ match_type! {
 		MultiLocation { parents: 1, interior: X1(Plurality { id: BodyId::Executive, .. }) }
 	};
 }
+match_type! {
+	pub type ParentOrSiblings: impl Contains<MultiLocation> = {
+		MultiLocation { parents: 1, interior: Here } |
+		MultiLocation { parents: 1, interior: X1(_) }
+	};
+}
 
 pub type Barrier = (
 	TakeWeightCredit,
 	AllowTopLevelPaidExecutionFrom<Everything>,
 	AllowUnpaidExecutionFrom<ParentOrParentsExecutivePlurality>,
 	// ^^^ Parent and its exec plurality get free execution
+	// Expected responses are OK.
+	AllowKnownQueryResponses<PolkadotXcm>,
+	// Subscriptions for version tracking are OK.
+	AllowSubscriptionsFrom<ParentOrSiblings>,
```

Comment:
@KiChjang would you be able to explain what a `Sibling` is in the context of XCM?

Reply:
No surprise here -- siblings are MultiLocations that have the same parent, as denoted by the definition `MultiLocation { parent: 1, junctions: X1(_) }`. It's equivalent to locating for a file under `../*`, where `..` is "up one level" (meaning parent) and "*" is the wildcard operator.

Reply:
Gotcha, thanks!